docs/Doku_PlanBuilder.md

```diff
@@ -0,0 +1,163 @@
+Documentation Plan Builder
```

Comment:
Please use markdown headings.

Please do not add "Documentation".

Do not use `Doku_` as filename.